testing/state-transition/state-transition.go

```diff
@@ -135,7 +138,7 @@ func SetupTestState(t *testing.T, cs chain.Spec) (
 	)
 
 	ctx := transition.NewTransitionCtx(
-		context.Background(),
+		ms,
```

Comment:
this is important because the state passed to `state_processor` is extract from an `sdk.Context`. If we use any ordinary `context.Context` tests would panic.